### PR TITLE
1 initial version of blocks module

### LINKED ISSUE #1: Initial version of blocks module

### PR DESCRIPTION
**Pull request by lnonce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## Install module
+
+- Open a Powershell terminal
+- `Install-Module buildingblocks` (from the Powershell Gallery)
+- `Import-Module buildingblocks`
+
+## Sponsors
+
+[![yendico AG](/assets/yendico_logo_textwhite_48.png)](https://yendico.ch)
```
Comment (svenborer) [inline README.md:18]: Assets are missing.

File: BuildingBlocks/BuildingBlocks.psd1 [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
+PrivateData = @{
+
+    PSData = @{
+
+        # Tags applied to this module. These help with module discovery in online galleries.
+        Tags = @("base64", "helpers", "retry")
+
+        # A URL to the license for this module.
+        LicenseUri = 'https://github.com/yendico-oss/building-blocks/blob/main/LICENSE.txt'
```
Comment (svenborer) [inline BuildingBlocks/BuildingBlocks.psd1]: Maybe rename project name to match module name?

**Review by svenborer — COMMENTED** [review]
Check my comments.

File: README.md (hunk 1 above)
Comment (lnonce) [reply README.md:18]: assets added

File: BuildingBlocks/BuildingBlocks.psd1 (hunk 2 above)
Comment (lnonce) [reply BuildingBlocks/BuildingBlocks.psd1]: url changed to https://github.com/yendico-oss/buildingblocks (no hyphen anymore)

**Review by svenborer — APPROVED** [review]
Looks good to me.

**Pull request merged**